flink-connectors/flink-sql-connector-hive-2.3.9/pom.xml

```diff
@@ -30,8 +30,8 @@ under the License.
 		<relativePath>..</relativePath>
 	</parent>
 
-	<artifactId>flink-sql-connector-hive-2.3.6_${scala.binary.version}</artifactId>
-	<name>Flink : Connectors : SQL : Hive 2.3.6</name>
+	<artifactId>flink-sql-connector-hive-2.3.9_${scala.binary.version}</artifactId>
```

Comment:
Renaming the module makes backporting difficult. I guess we only do it for 1.16?

Reply:
Yes, only for 1.16 since we've also only dropped the old Hive connectors in 1.16 as well